src/constants.ts

```diff
@@ -0,0 +1,48 @@
+import { StateType } from "./types";
+import {
+  generateN,
+  generateOneToN,
+  generateOutput,
+  generateStyles,
+} from "./Utils/functions";
+
+export const constraints = {
+  NO_OF_COLORS: 6,
+  NO_OF_CHANCES: 10,
+  NO_OF_GUESSES: 4,
+};
+
+export const COLORS = [
+  "gold",
+  "yellowgreen",
+  "aqua",
+  "pink",
+  "royalblue",
+  "orangered",
+];
+export const GUESSES: number[] = generateOneToN(constraints.NO_OF_GUESSES);
+export const NO_OF_CHANCES: number[] = generateOneToN(
+  constraints.NO_OF_CHANCES
+);
+
+export const INITIAL_STATE: StateType = {
+  selectedColor: 0,
```

Comment:
lets use enum instead on 0,1,2,3 for selected colors

Reply:
done